shared/rakefiles/entrypoint.rake

```diff
@@ -68,22 +66,15 @@ task :set_compose_env do
   end
 end
 
-desc "[ADVANCED] Authenticate and generate GCP credentials (gcloud auth login)"
-task :configure_login => [:set_vars] do
-  sh "#{@exekube_cmd} rake configure_login"
-end
-
-# This duplicates information in docker-compose.yaml,
-# TF_VAR_serviceaccount_key.
 desc "[ADVANCED] Create and download credentials for projectowner service account"
```

Comment:
Maybe this should say `[CI ONLY]` now?

Reply:
Agree. Updated comment and moved to `ci.rake`.